.npmignore

```diff
@@ -3,9 +3,14 @@
 .git
 .git*
 .git/*
+.github
 __tests__
-githooks
 coverage
+docs
+githooks
+test_openapi3.yml
+test_asyncapi.yml
 test_swagger.yml
-src
```

Comment:
src should be excluded from the node module

Reply:
alternatively you can simplify by adding files to package.json

```json
"files": [
    "README.md",
    "dist/"
]
```
Then only output files and the readme are included in the node_module